### DISCUSSION TITLE
删除新加模块的数据时无删除权限

### DISCUSSION THREAD

**Discussion by leezhang19** [zuihou/zuihou-admin-cloud#177] [02.提问]
已经给管理员授权了删除权限，但还是报错了
![image](https://user-images.githubusercontent.com/57745942/187358931-a996d686-54e7-4f66-bf05-edadefc9f2a1.png)
![image](https://user-images.githubusercontent.com/57745942/187359010-71a59ab5-2cda-48d1-ab2c-385ff4ccfb80.png)

**Comment by zuihou** [comment]
mysql.yml 或者 database.yml 里面 

lamp.database.isNotWrite = false

**Comment by zuihou** [comment]
这个问题跟“权限”无关， 跟 WriteInterceptor 有关。

WriteInterceptor 的目的是保护的演示环境数据不被人乱改

**Comment by zuihou** [comment]
> 

下次请说明使用的项目名、版本号等。

建议查看提问规范：https://github.com/zuihou/lamp-cloud/issues/175
